mrp_production_real_cost/README.rst

```diff
@@ -26,14 +26,15 @@ https://github.com/OCA/product-variant
 Usage
 =====
 
-Operating with a manufacture order, analytic entries adding costs will be
+Processing a manufacture order, analytic entries adding costs will be
 created when:
 
 * Some raw material is consumed.
 * A work order is finished or paused.
+* Also, together with *project_timesheet* module, users time is also translated
+to costs in the linked analytic account.
```

Comment:
When a user looks at the description of the `mrp_production_real_cost` module, it will be useful to see what is included in the cost. I think is more useful here. 
I'll add proper indentation.